packages/cli/core/src/lib/flags/platformCommonFlags.ts

```diff
@@ -9,7 +9,11 @@ import {
 export const withRegion = (withDefault = true) => ({
   region: Flags.string({
     char: 'r',
-    options: Object.keys(Region),
+    options: Object.keys(Region).flatMap((region) => [
```

Comment:
What if we called [Object.entries](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries) here? Would this remove a constraint/assumption that the region and value are the same aside from case?

Reply:
We still have to do a flatMap here, because otherwise you'd get a `string[][]`.

However it's a good idea to tie ourselves on both the value and the key instead of doing some processing (i.e. `toLowerCase`).

Reply:
https://github.com/coveo/cli/pull/919/files/958049fb2281199d6bf28bb2816d5eef3b98366c#r963916816